motoman_driver/src/industrial_robot_client/robot_state_interface.cpp

```diff
@@ -99,11 +98,7 @@ bool RobotStateInterface::init(SmplMsgConnection* connection)
     std::map<int, RobotGroup> robot_groups;
 
     std::string value;
-    if(!ros::param::search("topics_list", value))
```

Comment:
This (and the similar changes above) were committed recently.  You may need to rebase your changes.  See [here](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) for more info.